### PR TITLE
javascript-objects - Jeremy Nguyen

### PR DESCRIPTION
**Pull request by jeremyhnguyen**
Screenshot 2023-08-28 at 4 33 58 PM

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+console.log('value of student.livesInIrvine: ' + student.livesInIrvine);
+console.log(
+  'value of student.previousOccupation: ' + student.previousOccupation
+);
+console.log('value of student: ', student);
+
+const vehicle = { make: 'Tesla', model: 'Y', year: 2023 };
+
+vehicle['color'] = 'white';
+vehicle['isConvertible'] = 'no';
```
Comment (shawnkost) [inline javascript-objects/main.js]: The value for this should be a boolean rather than a string

File: javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+console.log('value of vehicle: ', vehicle);
+
+const pet = { name: 'Austin', type: 'dog' };
+
+delete pet.name;
+delete pet.type;
+
+console.log('value of pet: ' + pet);
```
Comment (shawnkost) [inline javascript-objects/main.js]: Do you see how in your console, you're getting `value of pet: [object, Object]`? This is exactly why we don't want to do console logs with the `+` operator, and we want to use a `,` to separate them instead. I will touch on this behavior in our Q&A.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change, and one comment here!

**Comment by jeremyhnguyen** [comment]
Fixed the boolean as well as the console log error, I forgot to switch to a comma for the last one after I went back through it.

Screenshot 2023-08-28 at 4 50 11 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**